src/helpers/db.ts

```diff
@@ -43,7 +43,5 @@ export const queryDatabase = async <T>(
 }
 
 export const isOkPacket = (o: any): o is OkPacket => {
-  return (
-    o && Object.prototype.hasOwnProperty.call(o, 'insertId') && typeof o.insertId === 'number' && o.affectedRows > 0
-  )
+  return o && Object.prototype.hasOwnProperty.call(o, 'insertId') && typeof o.insertId === 'number'
```

Comment:
```suggestion
  return 'insertId' in o && typeof o.insertId === 'number'
```
Do you know about the `in` operator?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

Reply:
Yep. Here I'm using `'insertId' in o` to make sure `insertId` exists as a property of that object.

Reply:
Oh wow, never seen this "Suggested change" feature, so I thought I was seeing a diff I made.

Ok, I'll give it a try.

Reply:
Updated, seems to work in the same way.